### ISSUE TITLE
what about integrate with webjack project ?

### ISSUE THREAD

**Issue by selevo** [orukusaki/avr-audio-bootloader#1]
https://webjack.io/

**Comment by orukusaki** [comment]
An interesting idea - the SoftModem module requires the Arduino headers so I doubt you'd be able to keep the image size under 1k.  The chip supports up to 4k though, so it should be possible to make it work.

**Comment by selevo** [comment]
yes, It would be very nice if the firmware could be updated directly from the browser page, the download code could be located directly on the page (loaded into the document body or simply inserted into the text field as hex), for example, you can make a virtual voltmeter / thermometer / motor controller on the page, or some other device that works in a pair of web page-arduino and a smartphone / computer is used as a display and control dashboard

thus it would be possible to abandon the LCD displays and control buttons, instead using a regular old phone that is connected to the Arduino through the  two-way audio channel.
 And primary information  transmits to the phone screen on the phone it was possible to do secondary math on and secondary information processing on js, then sending a subsequent command to the Arduino board to perform faster operations for example in real time.
